src/robot/running/model.py

```diff
@@ -175,6 +176,30 @@ def run(self, context, run=True, templated=False):
                 raise ReturnFromKeyword(self.values)
 
 
+@Body.register
+class Continue(model.Continue):
+    __slots__ = ['lineno']
+
+    def __init__(self, values=(), parent=None, lineno=None):
+        super().__init__(parent)
```

Comment:
`Continue`, and `Break`, shouldn't accept `values`.

Reply:
Corrected. Thought it will be empty so had kept. 

Removed